### PR TITLE
chore: Rename `hyperchain` to `subnet`

### LINKED ISSUE #159: Rename hyperchains to subnets throughout

### PR DESCRIPTION
**Pull request by gregorycoppola**
### Description
Does a global `s/hyperchain/subnet`.

### Applicable issues
- fixes #159

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,5 +48,5 @@ You can start with an NFT use case demo [here](https://github.com/hirosystems/st
 ## Resources
 
 - [Introductions to subnets.](https://www.youtube.com/watch?v=PFPwuVCGGuI)
-- [Introducing subnets blog post.](https://www.hiro.so/blog/introducing-hyperchains-by-hiro)
-- [Update on subnets, a scaling solution for Stacks.](https://www.hiro.so/blog/an-update-on-hyperchains-a-scaling-solution-for-stacks)
+- [Introducing subnets blog post.](https://www.hiro.so/blog/introducing-subnets-by-hiro)
+- [Update on subnets, a scaling solution for Stacks.](https://www.hiro.so/blog/an-update-on-subnets-a-scaling-solution-for-stacks)
```
Comment (obycode) [inline README.md:52]: These links haven't changed to subnets.

File: coverage.lcov [hunk 2]
```diff
@@ -0,0 +1,5595 @@
+TN:Ensure that L1 contract can't mint an NFT first created on the subnet if it already exists on the L1
```
Comment (obycode) [inline coverage.lcov:1]: I don't think this file should've been added.

File: README.md (hunk 1 above)
Comment (gregorycoppola) [reply README.md:52]: I should check all the markdown files

**Comment by a3slade** [comment]
If this gets merged, please close this: https://github.com/hirosystems/stacks-subnets/pull/162

**Comment by obycode** [comment]
Handled in #162

**Pull request closed without merging**